packages/@atjson/renderer-html/src/index.ts

```diff
@@ -282,6 +286,71 @@ export default class HTMLRenderer extends Renderer {
     }</div> <div style=" font-size: 15px; line-height: 21px; color: #000000; font-weight: 600; "> View on Threads</div></div></a></blockquote><script async src="https://www.threads.net/embed.js"></script>`;
   }
 
+  *InstagramEmbed(embed: Block<InstagramEmbed>) {
+    let slice =
+      embed.attributes.content != null &&
+      this.getSlice(embed.attributes.content);
+
+    return `<blockquote ${this.htmlAttributes({
+      id: embed.attributes.anchorName,
+      class: "instagram-media",
+      "data-instgrm-permalink": embed.attributes.url,
+      "data-instgrm-captioned": !embed.attributes.excludePostCaption,
+      "data-instgrm-version": "14",
+      style:
+        " background:#FFF; border:0; border-radius:3px; box-shadow:0 0 1px 0 rgba(0,0,0,0.5),0 1px 10px 0 rgba(0,0,0,0.15); margin: 1px; max-width:540px; min-width:326px; padding:0; width:99.375%; width:-webkit-calc(100% - 2px); width:calc(100% - 2px);",
+    }).join(" ")}><div style="padding:16px;"> <a ${this.htmlAttributes({
+      href: embed.attributes.url,
+      style:
+        " background:#FFFFFF; line-height:0; padding:0 0; text-align:center; text-decoration:none; width:100%;",
+      target: "_blank",
+    }).join(" ")}>${INSTAGRAM_SCAFFOLD}</a> <p ${this.htmlAttributes({
+      style:
+        " color:#c9c8cd; font-family:Arial,sans-serif; font-size:14px; line-height:17px; margin-bottom:0; margin-top:8px; overflow:hidden; padding:8px 0 7px; text-align:center; text-overflow:ellipsis; white-space:nowrap;",
+    }).join(" ")}> <a ${this.htmlAttributes({
+      href: embed.attributes.url,
+      style:
+        " color:#c9c8cd; font-family:Arial,sans-serif; font-size:14px; font-style:normal; font-weight:normal; line-height:17px; text-decoration:none;",
+      target: "_blank",
+    }).join(" ")}>${
+      slice ? this.render(slice) : `A post shared on Instagram`
+    }</a></p></div></blockquote> <script async src="//www.instagram.com/embed.js"></script>`;
+  }
+
+  *FacebookEmbed(embed: Block<FacebookEmbed>) {
+    let slice =
+      embed.attributes.content != null &&
+      this.getSlice(embed.attributes.content);
+
+    if (slice) {
+      return `<div ${this.htmlAttributes({
+        id: embed.attributes.anchorName,
+        class: "fb-post",
+        "data-href": embed.attributes.url,
+        "data-show-text": !embed.attributes.hideText,
+      }).join(" ")}><blockquote ${this.htmlAttributes({
+        class: "fb-xfbml-parse-ignore",
+        cite: embed.attributes.url,
+      }).join(" ")}>${this.render(slice)}</blockquote></div>`;
+    } else {
+      return `<iframe ${[
```

Comment:
Would it not make sense to still render the same structure as the above, just without the `<blockquote>` element (or with an empty `<blockquote>` element)?